[2024-02-21T08:13:03.635+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-19T01:00:00+00:00 [queued]>
[2024-02-21T08:13:03.665+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-19T01:00:00+00:00 [queued]>
[2024-02-21T08:13:03.665+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T08:13:03.697+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-19 01:00:00+00:00
[2024-02-21T08:13:03.709+0000] {standard_task_runner.py:60} INFO - Started process 170 to run task
[2024-02-21T08:13:03.717+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-19T01:00:00+00:00', '--job-id', '231', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp1f4iyq_6']
[2024-02-21T08:13:03.720+0000] {standard_task_runner.py:88} INFO - Job 231: Subtask get_mail
[2024-02-21T08:13:03.802+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-19T01:00:00+00:00 [running]> on host 968b17b6b013
[2024-02-21T08:13:03.933+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-19T01:00:00+00:00'
[2024-02-21T08:13:06.210+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/extract_email.py", line 16, in get_mail
    mail.authorize(account=os.getenv("GMAIL_ACCOUNT"), password=os.getenv("GMAIL_PASSWORD"))
  File "/home/airflow/.local/lib/python3.8/site-packages/RPA/Email/ImapSmtp.py", line 496, in authorize
    self.authorize_imap(
  File "/home/airflow/.local/lib/python3.8/site-packages/RPA/Email/ImapSmtp.py", line 453, in authorize_imap
    self.imap_conn.login(account, password)
  File "/usr/local/lib/python3.8/imaplib.py", line 603, in login
    raise self.error(dat[-1])
imaplib.IMAP4.error: b'[ALERT] Too many simultaneous connections. (Failure)'
[2024-02-21T08:13:06.225+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240219T010000, start_date=20240221T081303, end_date=20240221T081306
[2024-02-21T08:13:06.247+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 231 for task get_mail (b'[ALERT] Too many simultaneous connections. (Failure)'; 170)
[2024-02-21T08:13:06.300+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T08:13:06.327+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T08:18:10.742+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-19T01:00:00+00:00 [queued]>
[2024-02-21T08:18:10.754+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-19T01:00:00+00:00 [queued]>
[2024-02-21T08:18:10.755+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T08:18:10.766+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-19 01:00:00+00:00
[2024-02-21T08:18:10.772+0000] {standard_task_runner.py:60} INFO - Started process 130 to run task
[2024-02-21T08:18:10.774+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-19T01:00:00+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmp85s_m30w']
[2024-02-21T08:18:10.776+0000] {standard_task_runner.py:88} INFO - Job 266: Subtask get_mail
[2024-02-21T08:18:10.841+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-19T01:00:00+00:00 [running]> on host 9f5f399fdbe3
[2024-02-21T08:18:10.942+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-19T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-19T01:00:00+00:00'
[2024-02-21T08:18:13.655+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T08:18:17.330+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T08:18:17.337+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-289.txt')
[2024-02-21T08:18:17.344+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.346+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T08:18:17.352+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-290.txt')
[2024-02-21T08:18:17.353+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.354+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T08:18:17.359+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-291.txt')
[2024-02-21T08:18:17.359+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.360+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T08:18:17.364+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-292.txt')
[2024-02-21T08:18:17.365+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.366+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T08:18:17.369+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-293.txt')
[2024-02-21T08:18:17.369+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.370+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T08:18:17.371+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-294.txt')
[2024-02-21T08:18:17.372+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.372+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T08:18:17.374+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-295.txt')
[2024-02-21T08:18:17.375+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.375+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T08:18:17.377+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-296.txt')
[2024-02-21T08:18:17.378+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.378+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T08:18:17.379+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-297.txt')
[2024-02-21T08:18:17.379+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:17.380+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T08:18:17.381+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-298.txt')
[2024-02-21T08:18:17.382+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T08:18:18.233+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T08:18:18.239+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240219T010000, start_date=20240221T081810, end_date=20240221T081818
[2024-02-21T08:18:18.289+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T08:18:18.332+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
